### DISCUSSION TITLE
Looking to debug System.Net.Http.WinHttpHandler

### DISCUSSION THREAD

**Discussion by jensolsson** [dotnet/runtime#40767] [General]
I believe I spotted an issue with HTTP/2 in WinHttpHandler and want to investigate it further. I pulled the code and included System.Net.Http.WinHttpHandler.csproj in my solution but I am unable to compile and run it.

I get the following error in the log file from Visual Studio
=====================
2020-08-13 12:53:23
Recoverable
System.AggregateException: Project system data flow 'DataflowBlockSlim (ActionBlockSlimAsync`1 : 25115887)' closed because of an exception: System.AggregateException: One or more errors occurred. ---> System.ArgumentException: Must contain activeTargetFramework (Unsupported,Version=v0.0).
Parameter name: dependenciesByTargetFramework
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Snapshot.DependenciesSnapshot..ctor(String projectPath, ITargetFramework activeTargetFramework, ImmutableDictionary`2 dependenciesByTargetFramework)
 ...

I also get some errors regarding nuget

Severity | Code | Description | Project | File | Line | Suppression State
-- | -- | -- | -- | -- | -- | --
Error | CS0006 | Metadata file 'C:\runtime\artifacts\bin\System.Net.Http.WinHttpHandler\Debug\System.Net.Http.WinHttpHandler.dll' could not be found | ApnsTest | C:\Users\Jens Olsson\source\repos\ApnsTest\ApnsTest\CSC | 1 | Active
Error | NETSDK1004 | Assets file 'C:\runtime\artifacts\obj\System.Net.Http.WinHttpHandler\project.assets.json' not found. Run a NuGet package restore to generate this file. | System.Net.Http.WinHttpHandler | C:\Program Files\dotnet\sdk\3.1.301\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets | 234 |  
Error | NETSDK1004 | Assets file 'C:\runtime\artifacts\obj\System.Net.Http.WinHttpHandler\project.assets.json' not found. Run a NuGet package restore to generate this file. | System.Net.Http.WinHttpHandler | C:\Program Files\dotnet\sdk\3.1.301\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets | 234 |  
Error | NETSDK1004 | Assets file 'C:\runtime\artifacts\obj\System.Net.Http.WinHttpHandler\project.assets.json' not found. Run a NuGet package restore to generate this file. | System.Net.Http.WinHttpHandler | C:\Program Files\dotnet\sdk\3.1.301\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets | 234 |  

Is it possible to include WinHttpHandler like this? What am I doing wrong?

**Comment by danmoseley** [comment]
cc @dotnet/ncl
